tests/js/simple.js

```diff
@@ -0,0 +1,31 @@
+
+export const simple = async (fluvio, offset) => {
+  const topic = "simple-js-test";
+  const admin = await fluvio.admin();
+  try {
+    await admin.createTopic(topic, 1);
+  } catch(e) {
```

Comment:
Should we have a failure condition in the catch?

Reply:
I mostly did this because I wanted to rerun the unit test without deleting the topic. Now that I think about it, I think we might just have a setup, teardown and test function in each js test module.